mapproxy/test/system/test_arcgis.py

```diff
@@ -0,0 +1,62 @@
+# This file is part of the MapProxy project.
+# Copyright (C) 2010 Omniscale <http://omniscale.de>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import with_statement, division
+
+from io import BytesIO
+from mapproxy.request.arcgis import ArcGISRequest
+from mapproxy.test.image import is_png, create_tmp_image
+from mapproxy.test.http import mock_httpd
+from mapproxy.test.system import module_setup, module_teardown, SystemTest
+from nose.tools import eq_
+
+test_config = {}
+
+def setup_module():
+    module_setup(test_config, 'arcgis.yaml')
+
+def teardown_module():
+    module_teardown(test_config)
+
+transp = create_tmp_image((512, 512), mode='RGBA', color=(0, 0, 0, 0))
+
+class TestArcgisSource(SystemTest):
+    config = test_config
+    def setup(self):
+        SystemTest.setup(self)
+        # Doing the string joining here to handle the case of python27 and python3 rounding floats differently
```

Comment:
There is a bbox_aware_query_comparator option for this: https://github.com/mapproxy/mapproxy/blob/master/mapproxy/test/system/test_tms.py#L108